### DISCUSSION TITLE
Twig errors in entry uri format are ignored

### DISCUSSION THREAD

**Discussion by spharion88** [craftcms/cms#8669] [Ideas]
### Description
Adding invalid twig code to the entry URI leads to never changing URLs (without any error message). 

### Steps to reproduce
1. add a valid entry uri format to an entry (for example: "locations/{slug}")
2. view the entry uri in frontend
3. change the entry uri format to something invalid (for example: "locations/{{ someUndefinedObject }}/{slug}")
4. view the entry uri in frontend (it will be same as in step 2)

I would expect an exception (or any other error hint) in this case.

### Additional info

- Craft version: 3.1.25
- PHP version: 7.1.22
- Database driver & version: 5.7.23 - MySQL Community Server (MAMP 5.2.2)

**Comment by brandonkelly** [comment]
This is expected because we disable strict variables in Twig when rendering URI Format templates. I’ll mark as an enhancement request though, and we can consider making it possible to disable that behavior.

**Comment by spharion88** [comment]
A configuration option would be great to enable it on dev environments. At the moment it is really hard to detect twig errors in uri formats.
